### ISSUE TITLE
Site Editor: Ensure SnackBar (save confirmation message) is in the correct place

### ISSUE THREAD

**Issue by Marc-pi** [WordPress/gutenberg#47446]
### Description

In site editor (menu Appearance > Editor beta)

- Clicking on Save on the top right corner >> the confirmation message is on the bottom left !!!
- i'm sure we can improve this, should be displayed near by under the save button

please notice that we do not have the same behavior when editing a post (when we are not editing the templates)

![image](https://user-images.githubusercontent.com/919332/214811696-84c4a41d-463f-484d-b01a-07676e83f96a.png)
![image](https://user-images.githubusercontent.com/919332/214811786-abd6a6b4-119e-4b9e-a9a6-ece44f23ebba.png)
![image](https://user-images.githubusercontent.com/919332/214811834-7d95cfd8-73f5-4c0a-b251-fa5e6340c799.png)

---

i'm poking @jasmussen design team + @annezazu FSE Outreach program
declaread as bug, because we do not have the same behavior depending the admin area we are (site editor vs post editor)

### Step-by-step reproduction instructions

See above

### Environment info

- WP 6.1.1
- TT2 theme
- Gutenberg Blocks 15.0.1

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by ndiego** [comment]
> please notice that we do not have the same behavior when editing a post (when we are not editing the templates)

Hi @Marc-pi, thanks for reporting this. I want to confirm we see the same thing. When saving a post, I see a similar confirmation message as is displayed in the Site Editor. 

![save-behavior](https://user-images.githubusercontent.com/4832319/214955310-d9f90e44-83a4-4862-8986-e496115ae385.gif)

**Comment by annezazu** [comment]
I can replicate what Nick sees. Updating the issue title either way to make it easier to find.

**Comment by Marc-pi** [comment]
> When saving a post, I see a similar confirmation message as is displayed in the Site Editor.

* yep, my mistake, it is the same behavior (in site editor and update a post) 
* the confirmation message is displayed in the straight opposite side, which is a non sense !!!
* the message box is very usefull because the most frequent subsequent action is to click on View Post, but you have to move the mouse to cross the whole screen and reach the msg box modal
* moreover, you have additionally to right click on "open to new tab" to see the change you made, otherwise you lose your page
    * Use Case : "i write my post, i want to see the final result / rendered content quickly, and be able to fine tune my content or fix some errors like spelling" 

so you have 2 extra frictions !!!

**Comment by annezazu** [comment]
This matches what's been reported here: https://github.com/WordPress/gutenberg/issues/19749 Mind if I close this out as a result?

**Comment by Marc-pi** [comment]
+ yes, it explains my mistake : there is indeed a different behavior between publish and update, that's the right difference
+ target blank missing
+ confirmation is displayed in the opposite side (your eyes / mouse have to cross the screen to see the msg despite the action is on the opposite side) > not a good UX practice

i'll backport the points in the related post, to minimise the nr of tickets, so i close

**Comment by Marc-pi** [comment]
reopened, since here it deals with the confirmation box that is displayed at the opposite side of the screen than the publish button it is generated from.  #19749 is related and also quite different

>>>> the (clickable) snackbar msg is not located in proximity to the button that generates it (see https://core.trac.wordpress.org/ticket/40822), which is not natural (UX)

**Comment by westonruter** [comment]
> moreover, you have additionally to right click on "open to new tab" to see the change you made, otherwise you lose your page
> * Use Case : "i write my post, i want to see the final result / rendered content quickly, and be able to fine tune my content or fix some errors like spelling"

When opening the frontend page in a new tab, let's make sure the link uses a named window, the same as the "View preview" snackbar notification. See https://github.com/WordPress/gutenberg/issues/28479.
